solutions/silver/usaco-835.mdx

```diff
@@ -4,6 +4,25 @@ source: USACO Silver 2018 Open
 title: Lemonade Line
 author: Mrinall Umasudhan, Ryan Chou
 ---
+<Spoiler title="Hint 1">
+
+Consider how you can process the cows to minimize the number of cows joining the line. Think about that possible situations where a cow would be willing to wait behind the others.
```

Comment:
a bit vague?

Reply:
how specific do you want the hints to be? i think i did that intentionally to get them to think and not give much away